### PR TITLE
Implement PR testing for SUSE-Cloud/cct

### PR DESCRIPTION
**Pull request by dirkmueller**
Since cct is an integral part of the mkcloud gating run, we
also should run the selftest against cct PRs.

### CODE REVIEW

File: scripts/jenkins/ci.suse.de/openstack-mkcloud-pr-trigger.xml [hunk 1]
```diff
@@ -65,32 +60,39 @@ case $mode in
...
+for repo in ${cloud_repos[@]}; do
+    for github_pr in $($ghs -r $repo -a $action); do
+        github_pr_opts=(${github_pr//:/ })
+        github_pr_sha=${github_pr_opts[1]}
+        github_pr_branch=${github_pr_opts[2]}
+
+        if [ "$github_pr_branch" = "master" ]; then
+            # default config / may be overridden by tags from PR message
+            mkcloudtarget=all_noreboot
+            cloudsource=develcloud5
```
Comment (aspiers) [inline scripts/jenkins/ci.suse.de/openstack-mkcloud-pr-trigger.xml]: Why not `develcloud6`?

**Comment by aspiers** [comment]
There are more changes in this PR than the commit message describes.  For example why do you remove that Jenkins parameter?

File: scripts/jenkins/ci.suse.de/openstack-mkcloud-pr-trigger.xml (hunk 1 above)
Comment (dirkmueller) [inline scripts/jenkins/ci.suse.de/openstack-mkcloud-pr-trigger.xml]: I don't know why we decided to pick develcloud5 (also without TESTHEAD=1 set). but this is just code reindentation, I didn't want to change this in this commit..

**Comment by dirkmueller** [comment]
@aspiers because it doesn't really work anymore with the change to trigger more than automation repo PRs..

**Comment by aspiers** [comment]
Maybe in another PR we can introduce a new parameter which accepts a comma-separated list of values of the form `ORG/REPO#PR`.

**Comment by dirkmueller** [comment]
:shipit:

**Pull request merged**